### PR TITLE
Map Gestures - Fix briefing mode group

### PR DESCRIPTION
**Pull request by smitt14ua**
Fix error `Error append: Type Group, expected Array`.

### CODE REVIEW

File: addons/map_gestures/functions/fnc_getProximityPlayers.sqf [hunk 1]
```diff
@@ -37,7 +37,7 @@ switch (_this) do {
         _proximityPlayers append allUnits;
     };
     case (1): { // Players in Group
-        _proximityPlayers append group ace_player;
+        _proximityPlayers append units group ace_player;
```
Comment (jonpas) [inline addons/map_gestures/functions/fnc_getProximityPlayers.sqf:40]:
```suggestion
        _proximityPlayers append (units group ace_player);
```

Comment (veteran29) [reply addons/map_gestures/functions/fnc_getProximityPlayers.sqf]: `units` works also with just the unit.
`_proximityPlayers append (units ace_player);`

Comment (smitt14ua) [reply addons/map_gestures/functions/fnc_getProximityPlayers.sqf]: > `units` works also with just the unit.
> `_proximityPlayers append (units ace_player);`

Sounds good.

**Review by PabstMirror — APPROVED** [review]

**Review by veteran29 — APPROVED** [review]
Not a fan of these parentheses as I believe SQF is beautiful without them but LGTM

**Comment by bux** [comment]
> SQF is beautiful

...

File: addons/map_gestures/functions/fnc_getProximityPlayers.sqf [hunk 2]
```diff
@@ -37,7 +37,7 @@ switch (_this) do {
         _proximityPlayers append allUnits;
     };
     case (1): { // Players in Group
-        _proximityPlayers append group ace_player;
+        _proximityPlayers append (units ace_player);
```
Comment (commy2) [inline addons/map_gestures/functions/fnc_getProximityPlayers.sqf:40]:
```suggestion
        _proximityPlayers append units ace_player;
````

Comment (jonpas) [reply addons/map_gestures/functions/fnc_getProximityPlayers.sqf]: Why? Parentheses help readability without a single downside. It is used widely in the harshest standards to assure correct execution of code in the entire industry. I am not a fan of full MISRA for this use case, but this is a perfect example where parentheses outright improve readability (and possibly allow easy changing of commands that might have higher priority instead).

Comment (Dystopian) [reply addons/map_gestures/functions/fnc_getProximityPlayers.sqf]: I like commy's suggestion here. A matter of taste I think.

**Pull request merged**